lib/arena.rb

```diff
@@ -1,2 +1,39 @@
+require_relative "gladiator.rb"
+
 class Arena
-end
+  attr_accessor :name, :gladiators
+  def initialize(name)
+    @name = name.capitalize
+    @gladiators = []
+  end
+
+  def add_gladiator(gladiator)
+    if @gladiators.count < 2
+      @gladiators << gladiator
+      puts "#{gladiator.name} is in the arena"
+    end
+  end
+
+  def fight
+    if @gladiators.count == 2
+      puts "#{@gladiators[0].name} and his #{@gladiators[0].weapon} is fighting #{@gladiators[1].name} and his #{@gladiators[1].weapon}"
```

Comment:
One suggestion is to extract a couple variables: gladiator, gladiator2.  This code keeps comparing "stuff it grabs out an array".  I'd like to to be clearer that it's the same 2 gladiators, over and over. 

Then, when all tests are passing, play with this to see if you need all of these conditionals.  It's really easy to remove some and run tests.